### PR TITLE
only pay attention to the 'adbPort' opt

### PR DESCRIPTION
**Pull request by jlipps**
not the 'ANDROID_ADB_SERVER_PORT' env var

### CODE REVIEW

**Comment by jlipps** [comment]
looks like i broke it.

**Comment by jlipps** [comment]
retest this please

File: lib/adb.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ const DEFAULTS_OPTS = {
   javaVersion: null,
   suppressKillServer: null,
   jars: {},
-  helperJarPath: JAR_PATH
+  helperJarPath: JAR_PATH,
+  adbPort: 5037
```
Comment (imurchie) [inline lib/adb.js]: I'd prefer `5037` to be pulled out as a `const`, so it's obvious what the constants are.

**Comment by imurchie** [comment]
Looks good to me, other than the one comment.

**Comment by jlipps** [comment]
updated. however not sure how to get our e2e tests to rerun, any ideas?

**Comment by imurchie** [comment]
They ought to magically start. It takes a little while, though.

**Comment by jlipps** [comment]
ok tests passed. what do you think?

**Comment by imurchie** [comment]
Looks good to me. I think, though, that since the default port is referenced elsewhere (i.e., in android driver) it ought to be exported from this package.

**Comment by jlipps** [comment]
you're right, i was being lazy of course :-)

File: index.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 // transpile:main
 
 import ADB from './lib/adb';
+import { DEFAULT_ADB_PORT } from './lib/adb';
 
 export default ADB;
+export { DEFAULT_ADB_PORT };
```
Comment (imurchie) [inline index.js]: Cool!

I like the pattern, however, of putting the default into the named exports as well. That way if you just want the default you can use

```js
import ADB from 'appium-adb';
```

but if you want the default and something else you can use

```js
import { ADB, DEFAULT_ADB_PORT } from 'appium-adb';
```

and not 

```js
import { default as ADB, DEFAULT_ADB_PORT } from 'appium-adb';
```

**Comment by jlipps** [comment]
oops, i didn't realized you'd added a new comment and I merged. will add ADB to the named exports too

**Comment by jlipps** [comment]
fixed and published as 2.4.2

**Comment by jsdevel** [comment]
I have a need to pass in a different `adbPort` in capabilities, but it looks like it doesn't do anything right now.  Is there a willingness to add that as the `-P` param?

**Pull request merged**